src/decision/location_selection.jl

```diff
@@ -441,7 +441,11 @@ function selection_frequency(
 end
 
 """
-    selection_ranks(ranks::YAXArray{T, 3}, iv_type::Union{Symbol,Int64}; desc::Bool=true)::Vector{Int64}
+    selection_ranks(
```

Comment:
I thought the function signature in the docstring should be in one line.

Reply:
We don't follow that rule in other places 🤷 

Happy to replace with a single line though.